.github/workflows/publish.yml

```diff
@@ -57,3 +105,69 @@ jobs:
         with:
           password: ${{ secrets.PYPI_TOKEN }}
 
+      - name: Create a new release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_PERSONAL_ACCESS_TOKEN }}
+        with:
+          tag_name: ${{ inputs.release-version }}
+          release_name: Release ${{ inputs.release-version }}
+          draft: false
+          prerelease: false
+
+  build-dockers:
+    runs-on: ubuntu-latest
+    needs: [ publish ]
+    strategy:
+      fail-fast: false
+      matrix:
+        module:
+          - core
```

Comment:
Maybe we should have a list of modules to publish somewhere? instead of hard-coded here... And those should also be the modules running in the CI.
Not urgent for now, just an idea